### ISSUE TITLE
[SUGGESTION] Expand file handle limit to 8192

### ISSUE THREAD

**Issue by Sunconure11** [aers/EngineFixesSkyrim64#24]
According to some, this should be possible now, with some recent updates to windows, but no one has given me much info on it beyond that, so I’m kind of suggesting on a whim.

**Comment by aers** [comment]
A Windows 10 update sometime in the past year raised the cap from 2048 to 8192, so it's as simple as changing the function call. I'm not sure what happens if you try to set it that high on an OS that doesn't support it.

**Comment by Sunconure11** [comment]
What operating systems don't support it, and what percentage of players of Skyrim have those systems?

**Comment by Sunconure11** [comment]
Maybe have an experimental build with those changes, but not have it be the main build, to avoid screwing with players not using Windows?

**Comment by Sunconure11** [comment]
I also got word that someone did it for themselves in a personal use version.

**Comment by Sunconure11** [comment]
Hm, I created a fork with the change that I think should allow for this, at least, on a windows machine. How would I compile it?

https://github.com/Sunconure11/EngineFixesSkyrim64

**Comment by Sunconure11** [comment]
Sorry if I sound annoying, but some info on how to do this would be nice, since I’m trying to create a personal build for now.

Side note, would you be willing to take in the minor change, but for a different branch?

**Comment by Sunconure11** [comment]
I create a PR for this.

https://github.com/aers/EngineFixesSkyrim64/pull/27

**Comment by picobyte** [comment]
In Linux the default nr of open files can be read with `ulimit -n`, for me it is 1024, and can be set using `sudo ulimit -n 8192`. Mac seems to have the same command.

**Comment by JadeRae** [comment]
tl;dr If 2048 isn't high enough you can replace 'MaxStdio = 2048' to 'MaxStdio = 8192' in the EngineFixes.toml doc that comes with SSE Engine Fixes Pt. 1 to (safely?) increase your file handle limit.

Realize this post is a bit dated but wanted to stick an update on here for those who - like me - are running so many mods that 2048 file handle limit isn't high enough.  I started getting the save game error + data missing in documents and was enable to speak with certain npcs. Searched everywhere until I found that the fix was supposedly already in the SSE Engine Fixes mod (Which I've already had in my load order for some time.)

Stumbled upon this thread and decided to give it a whirl. Sure enough, I replaced the line in the EngineFixes.toml with 8192 and the game has been running smooth ever since. No fake corrupted saves, everyone will talk to me again, the world is right.

I'm running an up-to-date win 10, and nothing crazy as far as hardware goes. Got Skyrim running smooth with 1710 mods (198 esp+esm).

Oh, and yes. I have a small Skyrim addiction lol.
